### PR TITLE
Add Energy Australia

### PR DESCRIPTION
**Pull request by fletcherdodd**

### CODE REVIEW

File: entries/e/energyaustralia.com.au.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "Energy Australia": {
+    "domain": "energyaustralia.com.au",
+    "tfa": [
+      "email"
+    ],
+    "documentation": "https://www.energyaustralia.com.au/home/help-and-support/faqs/multi-factor-authentication",
+    "categories": [
+      "utilities"
+    ]
```
Comment (hkamran80) [inline entries/e/energyaustralia.com.au.json:14]:
```suggestion
    ],
    "regions": [
      "au"
    ]
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]
I can't find an actual SVG of their logo, so maybe extract the PNG from the SVG and just add that?

**Comment by fletcherdodd** [comment]
> I can't find an actual SVG of their logo, so maybe extract the PNG from the SVG and just add that?

I removed the reference image in Illustrator and created a new path same as the image it was referencing. Everytime I export the SVG add adds the image reference back. So I'll export it as a PNG

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
6KiB is a bit much for an icon :grimacing:
Try this 64x64px icon instead:
energyaustralia com au

**Review by hkamran80 — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**